src/main/kotlin/net/atos/zac/app/zaak/ZaakRestService.kt

```diff
@@ -314,6 +314,11 @@ class ZaakRestService @Inject constructor(
         with(policyService.readZaakRechten(zaak)) {
             assertPolicy(wijzigen && verlengenDoorlooptijd)
         }
+        with(restZaakEditMetRedenGegevens.zaak) {
+            if (behandelaar?.id != null && groep?.id != null) {
+                identityService.checkIfUserIsInGroup(behandelaar!!.id, groep!!.id)
+            }
+        }
```

Comment:
```suggestion
     restZaakEditMetRedenGegevens.zaak.run {
    behandelaar?.id?.let { behandelaarId ->
        groep?.id?.let { groepId ->
            identityService.checkIfUserIsInGroup(behandelaarId, groepId)
        }
    }
}
```

I would prefer this. Typically using `!!` should be avoided (except in test code here and there).

Reply:
Changed